.travis.yml

```diff
@@ -87,6 +87,7 @@ matrix:
         - make -j2 check
         - ( cd test/depstest && python3 depstest.py ../../../source/ )
         - ( cd .. && source/test/hdrtst/testtagsguards.sh )
+        - make dist # not sure if this is -j safe everywhere.
```

Comment:
does travis let you download artifacts?

Reply:
not sure. We could push them somewhere (like a github release) though.

Reply:
fwiw, you can publish artifacts from the pipelines.
ex: https://github.com/unicode-org/icu/blob/master/.ci-builds/.azure-pipelines.yml#L108
(This is how the windows binaries are produced.)
It should work for *nix as well, so we might be able to have a sort of rolling/evergreen make dist available.

Reply:
It looks like Travis can potentially directly deploy to a GitHub release:
https://docs.travis-ci.com/user/build-stages/deploy-github-releases/
which is definitely nicer.
AFAIK, the pipelines just give you an arbitrary file you can download. (So you'd have to copy/paste to a release).

Reply:
right. But I'm not sure what release it would go to.